src/redux/reducers/Dashboard/index.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-param-reassign */
+import {
+    LOADING_DASHBOARD_DATA,
+    STOP_LOADING_DASHBOARD_DATA,
+    LOAD_DASHBOARD_DATA,
+} from '../../actions/Dashboard/index';
```

Comment:
there is not need to put index, when you put the path as:
```'../../actions/dashboard'```
it will take the index as default file